### ISSUE TITLE
Validation set part of training set?

### ISSUE THREAD

**Issue by aelmiger** [nsantavas/Attention-A-Lightweight-2D-Hand-Pose-Estimation-Approach#2]
https://github.com/nsantavas/Attention-A-Lightweight-2D-Hand-Pose-Estimation-Approach/blob/2136e93586f0d0eb40518f47d2b9f78c6220ea33/model_train.py#L32-L33

Perhaps I am missing something, but why is your validation set part of the training set?
